Tests/SwiftProtobufTests/Test_JSON.swift

```diff
@@ -273,14 +271,9 @@ class Test_JSON: XCTestCase, PBTestHelpers {
         assertJSONDecodeSucceeds("{\"singleInt64\": \"0\" }") {$0.singleInt64 == 0}
         // Protobuf JSON does accept exponential format
         assertJSONDecodeSucceeds("{\"singleInt64\":1e3}") {$0.singleInt64 == 1000}
-        assertJSONDecodeSucceeds("{\"singleInt64\":9.223372036854775807e18}") {$0.singleInt64 == Int64.max}
-        assertJSONDecodeSucceeds("{\"singleInt64\":9.223372036854775807e+18}") {$0.singleInt64 == Int64.max}
-        assertJSONDecodeSucceeds("{\"singleInt64\":-92.23372036854775808e17}") {$0.singleInt64 == Int64.min}
-        assertJSONDecodeSucceeds("{\"singleInt64\":9223372036854775807000e-3}") {$0.singleInt64 == Int64.max}
-        assertJSONDecodeSucceeds("{\"singleInt64\":9223372036854775807000000000e-9}") {$0.singleInt64 == Int64.max}
```

Comment:
Do you mean a quoted FP literal or normalized to integer form and quoted?

As they're written now, the 1st, 3rd, and 4th ones have too many significant digits to fit in a `double`, quoted or not, so they were no longer valid tests. The 2nd one still passes, but it isn't significantly different from another test that uses the same number shifted one decimal place over. Since the implementation has been converted to just use `Double(_: String)`, I didn't think it was worth it for us to essentially be testing Swift's stdlib's double-parsing, and the conformance tests cover this more thoroughly.